app/services/invite_schools.rb

```diff
@@ -134,6 +134,75 @@ def send_induction_coordinator_sign_in_chasers
     end
   end
 
+  def send_induction_coordinator_choose_route_chasers
+    induction_coordinators_not_chosen_route = User.includes(:induction_coordinator_profile, schools: :school_cohorts)
+                                                  .joins(:induction_coordinator_profile)
+                                                  .left_outer_joins(schools: :school_cohorts)
+                                                  .where(school_cohorts: { id: nil })
+                                                  .where.not(last_sign_in_at: nil)
+    induction_coordinators_not_chosen_route.find_each do |induction_coordinator|
+      school_without_cohort = induction_coordinator.schools
+                                                   .first { |school| school.school_cohorts.none? }
+      SchoolMailer.induction_coordinator_reminder_to_choose_route_email(
+        recipient: induction_coordinator.email,
+        name: induction_coordinator.full_name,
+        school_name: school_without_cohort.name,
+        sign_in_url: choose_route_chaser_sign_in_url,
+      ).deliver_later
+    rescue StandardError
+      logger.info "Error emailing induction coordinator, email: #{induction_coordinator.email} ... skipping"
+    end
+  end
+
+  def send_induction_coordinator_choose_provider_chasers
+    induction_coordinators_not_in_partnership = User.includes(
+      :induction_coordinator_profile,
+      schools: %i[school_cohorts partnerships],
+    ).where(
+      id:
+        School.unpartnered(Cohort.current.start_year)
+              .joins(:school_cohorts, :induction_coordinator_profiles)
+              .where(school_cohorts: { induction_programme_choice: "full_induction_programme" })
+              .pluck(:user_id),
+    )
+
+    induction_coordinators_not_in_partnership.find_each do |induction_coordinator|
+      school_without_provider = induction_coordinator.schools.first do |school|
+        school.school_cohorts.first.induction_programme_choice == "full_induction_programme" &&
+          school.partnerships.none? { |partnership| partnership.challenge_reason.nil? }
+      end
```

Comment:
should we make it a `filter` rather than `first`? If a coordinator has many schools and hasn't chosen a partner for any of them, maybe they should get an email for each?

Reply:
First was a decision, some of these have 40 schools, we only want to email them once